client/pages/DashboardPage.react.js

```diff
@@ -3,14 +3,25 @@ import { connect } from 'react-redux';
 import { NavItem, Navbar, Nav } from 'react-bootstrap';
 import { signsOut, getsProject } from '../redux/actions';
 import { browserHistory } from 'react-router';
-import SidebarNavigation from '../components/dashboardPageComponents/sidebarNavigation/MyProjects';
+import SidebarNavigation from '../components/dashboardPageComponents/sidebarNavigation/SidebarNavigation';
 import Content from '../components/dashboardPageComponents/contentComponents/Content';
 
 export default class DashboardPage extends Component {
+  constructor (props) {
+    super(props);
+    this.state = {
+      projectModalVisibility : false
+    };
+  };
+
   handleLogout () {
     this.props.dispatch(signsOut(browserHistory));
   };
 
+  toggleProjectVisibility () {
+    this.setState({ projectModalVisibility : !this.state.projectModalVisibility });
```

Comment:
change to:

this.setState(prev => ({ projectModalVisibility : !prev.projectModalVisibility }));